eoscostarica/eoscostarica.io
# Refactor home ui

### GH Issue

- Refactor UI Home
- Refactor Navbar menu
- Add language selector

### Steps to test
1. run yarn start
1. go to localhost:3000
1. see the home page

#### CheckList
